lib/stat_tracker.rb

```diff
@@ -133,10 +133,10 @@ def all_coaches
    @game_teams.map {|game_team| game_team.head_coach}.uniq
   end
 
-  def create_hash_with_team_games_by_coach(season_id)
-     all_coaches.reduce({}) do |coaches_with_games, coach|
-       coaches_with_games[coach] = game_teams_that_season_by_coach(coach, season_id)
-       coaches_with_games
+  def game_teams_by_coach_for_season(season_id)
+    @game_teams.reduce(Hash.new { |h,k| h[k] = [] }) do |result, game_team|
+      result[game_team.head_coach] << game_team if game_team.season == season_id
+      result
     end
   end
 
```

Comment:
This method, `game_teams_that_season_by_coach` is no longer used right? I think you can delete it. Same goes with the `all_coaches` method.

Reply:
The code highlighting was acting weird so refer to the method names instead of the highlighted code you see here.

Reply:
Ahhh, my bad! Forgot to delete those guys. I'll fix it. Thank you!